app/components/notifier/modal-new-edit/component.js

```diff
@@ -160,4 +161,27 @@ export default Component.extend(ModalBase, NewOrEdit, {
     get(this, 'modalService').toggleModal();
   },
 
+  validate() {
+    this._super();
+    const errors = get(this, 'errors') || [];
+    const intl = get(this, 'intl')
+    const preError = '"Default Recipient" is required'
+
+    if (errors.includes(preError)) {
+      const notifierType = get(this, 'model.notifierType')
+      let afterError = ''
+
+      if (notifierType === 'slack') {
+        afterError = 'Default Channel'
+      }
+      if (notifierType === 'email') {
+        afterError = 'Default Recipient Address'
```

Comment:
You check this string multiple times in different files. Strings like this are good candidates for constants file.